.github/workflows/test_and_deploy.yml

```diff
@@ -2,8 +2,6 @@ name: tests
 
 on:
   push:
-    branches:
-      - main
```

Comment:
I claim this will make it even harder to solve #155 because then there'll be _no_ builds reporting the coverage from ``main``. It's good from a 🌱 perspective though. No need to duplicate server work.

Happy to approve as an experiment, but I'm still betting on my  ``if: ${{ !contains(github.ref, 'gh-readonly-queue/') }}`` as more likely to solve. (Unfortunately, any play tests have to go onto ``main``, by construction.)